apps/prairielearn/src/pages/partials/instructorInfoPanel.ejs

```diff
@@ -50,10 +50,23 @@
       <% } %>
     </div>
   </div>
-    <div class="d-flex flex-wrap">
-      <div class="pr-1">Title:</div>
-      <div><%= question.title %></div>
-    </div>
+
+  <% if (typeof question_is_shared !== 'undefined' && question_is_shared) { %>
+  <div class="d-flex flex-wrap">
+    <div class="pr-1">Shared As:</div>
+    <% if (question.shared_publicly) { %>
+      <div>
+        <a href="<%= plainUrlPrefix %>/public/course/<%= course.id %>/question/<%= question.id %>/preview?variant_seed=<%= variant.variant_seed %>">@<%= course.sharing_name %>/<%= question.qid %></a>
```

Comment:
do we care to preserve the variant see here or no? 

It makes sense when linking back to the same page you are already on, but when linking from the private instructor preview, to the public preview, do we want it?

Reply:
I'm inclined to not preserve the variant seed.

Reply:
done